test-support/helpers/ember-tooltips.js

```diff
@@ -105,37 +184,39 @@ export function assertTooltipVisible(assert, options={}) {
 export function assertTooltipSide(assert, options = {}) {
   const { side } = options;
 
-  const sideIsValid = (
-    side === 'top' ||
-    side === 'right' ||
-    side === 'bottom' ||
-    side === 'left'
-  );
+  validateSide(side);
 
-  /* We make sure the side being tested is valid. We
-  use Ember.assert because assert is passed in from QUnit */
+  const { expectedGreaterDistance, expectedLesserDistance } = getPositionDifferences(options);
 
-  if (!sideIsValid) {
-    Ember.assert(`You must pass side like assertTooltipSide(assert, { side: 'top' }); Valid options for side are top, right, bottom, and left.`);
-  }
+  /* When the side is top or left, the greater number
+  is the target's position. Thus, we check that the
+  target's position is greater than the tooltip's
+  position. */
 
-  const expectedSide = options.side || 'top';
-  const $target = getTooltipTargetFromBody(options.targetSelector);
-  const targetPosition = $target[0].getBoundingClientRect();
-  const $tooltip = getTooltipFromBody(options.selector);
-  const tooltipPosition = $tooltip[0].getBoundingClientRect();
+  assert.ok(expectedGreaterDistance > expectedLesserDistance,
+    `Tooltip should be on the ${side} side of the target`);
+}
+
+export function assertTooltipSpacing(assert, options) {
+  const { side, spacing } = options;
 
-  if (expectedSide === 'top') {
-    assert.ok(targetPosition.top > tooltipPosition.bottom,
-      'Tooltip should be above the target');
-  } else if (expectedSide === 'right') {
-    assert.ok(targetPosition.right < tooltipPosition.left,
-      'Tooltip should be right of the target');
-  } else if (expectedSide === 'bottom') {
-    assert.ok(targetPosition.bottom < tooltipPosition.top,
-      'Tooltip should be below the target');
-  } else if (expectedSide === 'left') {
-    assert.ok(targetPosition.left > tooltipPosition.right,
-      'Tooltip should be left of the target');
+  validateSide(side, 'assertTooltipSpacing');
+
+  if (typeof spacing !== 'number') {
+    Ember.assert(`You must pass spacing as a number like assertTooltipSpacing(assert, { side: 'top', spacing: 10 });`);
   }
+
+  const { expectedGreaterDistance, expectedLesserDistance } = getPositionDifferences(options);
+  const actualSpacing = Math.round(expectedGreaterDistance - expectedLesserDistance);
```

Comment:
do you need to allow for a margin of error like before?

https://github.com/sir-dunxalot/ember-tooltips/pull/145/files#diff-b7eb48ff1a009e6b663bff11a464e390L15